### ISSUE TITLE
I am working on a awesome-python-htmx, seeking your feedback

### ISSUE THREAD

**Issue by benjamin-kirkbride** [pcarbonn/fast_html#1]
A few of us at PyCon this year got together and brainstormed a new Web Stack that we are calling PyHAT (Python, htmx, ASGI, TailwindCSS). The first thing we set out to do is create [awesome-python-htmx](https://github.com/PyHAT-stack/awesome-python-htmx/blob/main/README.md); a collection of active tools/libraries/projects in that space.

Your project mentions htmx, which is how I found it. That said, it's not extremely clear to me exactly what benefit it provides over something like Jinja, specifically in the context of htmx.  I'd appreciate it if you have any additional context that you could provide, assuming you have interest in your tool being included.

In addition to that, you could also participate in https://github.com/PyHAT-stack/awesome-python-htmx/discussions/1 if you like.

**Comment by pcarbonn** [comment]
I developed fast-html for a Python/htmx project I have.  The main benefit is that you get the full capability of the Python language and that you do not need to learn a special templating language, which is often limited in its capabilities.

Note verybody agrees, though.  See [the discussion on reddit](https://www.reddit.com/r/Python/comments/xsqnn5/fast_html_a_fast_minimalist_html_generator/) about it 6 months ago.  Still, I would not want to use Jinja.

**Comment by benjamin-kirkbride** [comment]
Is the project you are working on OSS? I'd love to see it if it is :)

**Comment by pcarbonn** [comment]
Yes, it is OSS.  The project is IDP-Z3 <http://www.idp-z3.be>.  The website
in htmx is here <https://interactive-consultant.idp-z3.be/htmx>.  The htmx
code is here
<https://gitlab.com/krr/IDP-Z3/-/blob/main/idp_web_server/htmx.py>.

Thank you for your interest.
PC

**Comment by benjamin-kirkbride** [comment]
I have included it in there :)
